source/includes/crud/Insert.java

```diff
@@ -31,7 +34,24 @@ public static void main(String[] args) {
                         .append("genres", Arrays.asList("Documentary", "Comedy")));
 
                 // Prints the ID of the inserted document
-                System.out.println("Success! Inserted document id: " + result.getInsertedId());
+                System.out.println("InsertOne document id: " + result.getInsertedId());
```

Comment:
S: same suggestion as above; applies to all
```suggestion
                System.out.println("insertOne() document id: " + result.getInsertedId());
```